.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Python CI
```

Comment:
Would it make sense to add `commitlint.yml` too? It would look like this:
```yaml
# Run commitlint on the commit messages in a pull request.

name: Lint Commit Messages

on:
  - pull_request

jobs:
  commitlint:
    uses: openedx/.github/.github/workflows/commitlint.yml@master
```

Reply:
Added.